apps/communications/contacts/test/unit/views/load/load_controller_test.js

```diff
@@ -0,0 +1,102 @@
+/* global LazyLoader, ContactsService, MockMatcher, MockContactsList */
+/* global LoadController, utils */
+
+'use strict';
+
+require('/shared/js/contacts/import/utilities/status.js');
+requireApp('communications/contacts/test/unit/mock_contacts_match.js');
+requireApp('communications/contacts/services/contacts.js');
+requireApp('communications/contacts/test/unit/mock_contacts_list.js');
+require('/shared/js/lazy_loader.js');
+requireApp('communications/contacts/views/load/js/load_controller.js');
+
+suite('LoadController', function() {
+
+  var stubLazyLoader;
+  var stubContactServiceSave;
+  var stubUtilsStatus;
+  var contacts;
+  var realMatcher;
+  var fakeActivity = {
+    source: {
+      data: {
+        allowSave: true,
+        blob: null,
+        filename:
+          '/apps/communications/contacts/test/unit/vcards/vcard_21.vcf',
+        type:'text/x-vcard'
+      }
+    },
+    postResult: function foo() {}
+  };
+
+  suiteSetup(function() {
+    LoadController.init();
+    LoadController.setActivity(fakeActivity);
+    stubLazyLoader = sinon.stub(LazyLoader, 'load', function(files, callback) {
```

Comment:
if you use `this.sinon.stub` on `setup` instead, then there's no need of restoring it later.